app/migrations/20191226170840_import_version_groups.js

```diff
@@ -0,0 +1,60 @@
+const path = require("path");
+const { getDirectoryContent, extractIdFromUrl } = require("./utils");
+
+const VERSION_GROUP_DIR = path.join(__dirname, "../../data/version-group");
+
+const createVersionGroupTable = async knex =>
+  knex.schema
+    .createTable("version_group", table => {
+      table
+        .integer("id")
+        .unsigned()
+        .notNullable()
+        .primary();
+    })
+    .alterTable("version", table => {
+      table.integer("version_group_id").unsigned();
+      table.foreign("version_group_id").references("version_group.id");
+    });
+
+const mapVersionGroupToVersionGroupDatabase = versionGroup => ({
+  id: versionGroup.id
+});
+
+const insertAllVersionGroups = async knex => {
+  const allVersionGroups = await getDirectoryContent(VERSION_GROUP_DIR);
+
+  for (const versionGroups of allVersionGroups) {
+    const versionGroupsDatabase = mapVersionGroupToVersionGroupDatabase(
+      versionGroups
+    );
+
+    console.log("Importing version group: ", versionGroupsDatabase.id);
+    await knex.insert(versionGroupsDatabase).into("version_group");
+
+    const versionIds = versionGroups.versions
+      .map(x => x.url)
+      .map(url => extractIdFromUrl("version", url));
+
+    for (const versionId of versionIds) {
+      await knex("version")
+        .where({ id: versionId })
+        .update({ version_group_id: versionGroupsDatabase.id });
+    }
+  }
+};
+
+exports.up = async knex => {
+  await createVersionGroupTable(knex);
+  await insertAllVersionGroups(knex);
+};
+
+exports.down = knex =>
+  knex.schema
+    .alterTable("version", table => {
+      // This is not supported for SQLite.
+      // Comment the line below for rollbacking SLIte DBs.
```

Comment:
It'd be nice when we have time to write these instructions in a `CONTRIBUTING.md` file. It would allow to gather the knowledge in a single file.

```suggestion
      // Comment the line below for rolling back SQLIte DBs.
```

Reply:
Sure things but I think this PR is not the place to introduce this file. We can work on it later but you're right 👍